corehq/messaging/smsbackends/telerivet/tests/test_log_call.py

```diff
@@ -1,10 +1,10 @@
-from corehq.apps.ivr.tests.util import LogCallTestCase
+import corehq.apps.ivr.tests.util as util
```

Comment:
That is odd.

On 29 February 2016 at 15:35, Daniel Miller <notifications@github.com>
wrote:

> In corehq/messaging/smsbackends/telerivet/tests/test_log_call.py
> <https://github.com/dimagi/commcare-hq/pull/10522#discussion_r54409343>:
>
> > @@ -1,10 +1,10 @@
> > -from corehq.apps.ivr.tests.util import LogCallTestCase
> > +import corehq.apps.ivr.tests.util as util
>
> I don't understand it really, but for some reason the old import was
> causing LogCallTestCase to appear as if it was defined inside this
> module, which was causing this conditional
> <https://github.com/dimagi/commcare-hq/blob/a7963baf03e7de2f2124de7c7d229487a9338202/corehq/apps/ivr/tests/util.py#L55>
> to fail.
>
> see also: commit message on a7963ba
> <https://github.com/dimagi/commcare-hq/commit/a7963baf03e7de2f2124de7c7d229487a9338202>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/10522/files#r54409343>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa